packages/config/src/layer2s/rhinofi.ts

```diff
@@ -93,12 +93,7 @@ export const rhinofi: Layer2 = {
     dataAvailability: {
       ...RISK_VIEW.DATA_EXTERNAL_DAC,
       sources: [
-        {
-          contract: 'StarkExchange',
-          references: [
-            'https://etherscan.io/address/0x1c3A4EfF75a287Fe6249CAb49606FA25659929A2#code#F34#L183',
-          ],
-        },
+        // TODO: add reference to new StarkExchange implementation!
```

Comment:
Should this stay here?

Reply:
yes, as i said in the PR description rhino (= deversifi) upgraded the contracts and i will address it in another PR, that reference has been removed otherwise it breaks the build and has been replaced with a TODO